.travis.yml

```diff
@@ -2,11 +2,12 @@ os:
   - linux
```

Comment:
If you remove the os line and replace it with:
```
dist: jammy
```
to use the latest version of ubuntu rather than the default which is 6 years old using node js 18 or the normal lts option should work perfectly fine

Reply:
I updated the travis CI build script to use ubuntu 22.04 which is the latest LTS release of ubuntu which includes a newer version of the glibc library that node.js 18 requires

Reply:
Nice catch, thanks :)